src/ipyaladin/region_converter.py

```diff
@@ -0,0 +1,191 @@
+import numpy as np
+
+from astropy.coordinates import SkyCoord, CartesianRepresentation, Angle
+from astropy.coordinates.matrix_utilities import rotation_matrix
+
+try:
+    from regions import (
+        RectangleSkyRegion,
+        PolygonSkyRegion,
+        Region,
+        CircleSkyRegion,
+        EllipseSkyRegion,
+        LineSkyRegion,
+    )
+except ImportError:
+    RectangleSkyRegion = None
+    PolygonSkyRegion = None
+    Region = None
+    CircleSkyRegion = None
+    EllipseSkyRegion = None
+    LineSkyRegion = None
+from typing import Union
+
+
+def rectangle_to_polygon_region(region: RectangleSkyRegion) -> PolygonSkyRegion:
+    """Convert a RectangleSkyRegion to a PolygonSkyRegion.
+
+    Parameters
+    ----------
+    region : RectangleSkyRegion
+        The region to convert.
+
+    Returns
+    -------
+    PolygonSkyRegion
+        The converted region.
+
+    """
+    bl_x, bl_y, bl_z = (
```

Comment:
could you keep coordinates triplets in tuples/lists?  I don't think the unpacked values are used here?